src/Command/CreateLocalAccountCommand.php

```diff
@@ -96,21 +96,23 @@ protected function execute(InputInterface $input, OutputInterface $output): int
     {
         $email = $input->getArgument('email');
         $name = $input->getArgument('name');
+        assert(is_string($name));
         $pass = $input->getArgument('pass');
+        assert(is_string($pass));
```

Comment:
Please group multiple related asserts together, eg.
```php
assert(is_string($name));
assert(is_string($pass));
```
or maybe even
```php
assert(is_string($name) && is_string($pass));
```

Reply:
while I do agree with grouping based on logic, I did choose for exactly that reason to group them like this. It might be a little premature, but there is no reason to even ask a pass if `$name` isn't a string.

While the code is pretty short and relatively close to each other, it prefer to have kiwi fail when we're in an undesirable state (like not having a valid `$name`. And not getting in the habit of proceeding with said state.

Reply:
I feel like, given the code structure presented, that this reasoning is based upon a premature optimization. Exiting an invalid state at the earliest opportunity is only necessary when performance dictates its necessity; otherwise, its weight as an argument should be discarded when legibility is affected (which is always the case).

To clarify, it doesn't mean that I feel like we should _never_ exit early. For example, a `getArgumentAsStr(Input $input, string $name): string;` function would allow both early exits and increased legibility (resembling the code structure of a parser). However, when this specific type of code structure is absent, it is more common practice in general to use a pipelined approach to structure code. In this context, that would mean to (1) bind all data and then (2) validate it. For example, see how [the constructor for the Symfony Input class](https://github.com/symfony/console/blob/6.2/Input/Input.php#L36) is structured, first binding all data and then validating it.

Now, we're talking of course about two lines of code, so the impact of any approach is minimal. However, I wanted to give this more expanded response because I feel like we should be wary of premature optimizations of Kiwi code in a general sense. Let me know what you think.